### DISCUSSION TITLE
Wazuh agent FIM on alternate drives that are not C:\

### DISCUSSION THREAD

**Discussion by DinhDiesel** [Security-Onion-Solutions/securityonion#6095] [Unsupported Versions] [answered]
I have tested wazuh FIM with local folder on C:\user\who\file.txt, I am able to get data but when I add lets say the D:\file-shares (I cant get data).

**Comment by dougburks** [comment, marked as answer]
Have you checked the Wazuh agent log for additional clues?

**Reply by DinhDiesel** [reply]
i just did recently and i get - "WARNING: (6914): Cannot create a snapshot of file 'e:\file, and  ERROR: in w_compress_gzfile(): gzopen error queue/dif/local/e\ .... so the line i added to ossec.conf was <directories check_all="yes" whodata="yes" report_changes="yes>E:\folder\files</directories>. Seeing that i took out report_changes="yes". Testing now see if that was the issue.

**Reply by DinhDiesel** [reply]
for anyone that is looking at this, whodata and realtime are for critical files, having this for directories with alot of files is not best practice too many logs to process. and if you use whodata that does use realtime monitoring also (which i did not do lol).
